packages/@okta/i18n/src/properties/login.properties

```diff
@@ -833,24 +833,17 @@ oie.custom_otp.verify.passcode.label = Enter code
 oie.on_prem.authenticator.description = Verify by entering a code generated by {0}.
 oie.on_prem.authenticator.default.vendorName = Custom On-prem Authenticator
 oie.on_prem.enroll.title = Set up {0}
-oie.on_prem.enroll.username.label = Enter authenticator username
-oie.on_prem.enroll.passcode.label = Enter authenticator passcode
 oie.on_prem.verify.title = Verify with {0}
-oie.on_prem.verify.passcode.label = Enter code
```

Comment:
For the string `Enter code` we have created oie specifc keys 
oie.custom_otp.verify.passcode.label, oie.google_authenticator.otp.enterCodeText should they be replaced with `mfa.challenge.enterCode.placeholder` as well ?

Reply:
I see that we are preserving any V1 customization a customer may have with this change where we are re-using `mfa.challenge.enterCode.placeholder` for on prem and rsa. 

But What if customer wanted to customize the string as 
```
Enter Rsa Code
Enter onPrem Code
```

With this change they wont be able to do that correct ? They can only do one customization for `mfa.challenge.enterCode.placeholder` that will show up in both authenticators.

Reply:
Resolved offline on slack